### ISSUE TITLE
Transcribe fails with "no attribute queue"

### ISSUE THREAD

**Issue by carlaKC** [bitcointranscripts/tstbtc#124]
Ran into an issue on: 04c3a82 (not present in f905534, so recently introduced)

```
(venv) ➜  tstbtc git:(main) tstbtc transcribe "PATH" --deepgram --diarize  --loc lightning-specification  -t 'Lightning Specification Meeting' -d  '2024-05-20' -T 'lightning' --noqueue
/Users/carla/Work/bitcointranscripts/tstbtc/venv/lib/python3.11/site-packages/whisper/timing.py:57: NumbaDeprecationWarning: The 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.
  @numba.jit
Traceback (most recent call last):
  File "/Users/carla/Work/bitcointranscripts/tstbtc/venv/bin/tstbtc", line 5, in <module>
    from transcriber import cli
  File "/Users/carla/Work/bitcointranscripts/tstbtc/venv/lib/python3.11/site-packages/transcriber.py", line 463, in <module>
    cli.add_command(commands.queue)
                    ^^^^^^^^^^^^^^
AttributeError: module 'app.commands' has no attribute 'queue'
```

**Comment by kouloumos** [comment]
I sometimes forget that there's another person out there using this tool, so it's great to hear from you. 😄

I had already addressed this issue in a feature branch (see [this commit](https://github.com/bitcointranscripts/tstbtc/commit/975fbf95bfc247a2e99d7f3ed02321b9da7d95ce)), but it’s going to take a bit of time before it merges with the main branch. In the meantime, I’ve cherry-picked the fix directly to the main branch in [this commit](https://github.com/bitcointranscripts/tstbtc/commit/61b3dfa581273d0d1a6a92a3c1ec679b75ffe989), so the problem should be resolved now!

**Comment by kouloumos** [comment]
Additional productivity tip: tstbtc now supports [a configuration file](https://github.com/bitcointranscripts/tstbtc?tab=readme-ov-file#configuration), so by creating the following `config.ini` you will not need to repeat those flags everytime:
```
deepgram = True
diarize = True
noqueue = True
```

**Comment by carlaKC** [comment]
> I sometimes forget that there's another person out there using this tool, so it's great to hear from you. 😄

Greetings from the abyss! And thank you for the quick fix 🫶  Confirmed fixed on 1845a6e.

> tstbtc now supports [a configuration file](https://github.com/bitcointranscripts/tstbtc?tab=readme-ov-file#configuration), so by creating the following config.ini you will not need to repeat those flags everytime:

Verrynice 👌
